Assets/_BForBoss/_Utility/Scripts/PostProcessingVolumeWeightTool.cs

```diff
@@ -3,11 +3,6 @@
 
 namespace Perigon.Utility
 {
-    /// <summary>
-    /// Tool to distort the lens.
-    /// WARNING/NOTE - Post Processing > Volume > LensDistortion - make sure intensity is turned on
-    /// For some reason there's no built in function to enable this automatically. 
```

Comment:
Is this warning something we don't need anymore?

Reply:
yeah, this is when the class did something else.